src/snowcli/cli/__init__.py

```diff
@@ -19,7 +19,10 @@
 from snowcli.config import AppConfig
 from snowcli.snowsql_config import SnowsqlConfig
 
-app = typer.Typer(context_settings={"help_option_names": ["-h", "--help"]})
+app = typer.Typer(
+    context_settings={"help_option_names": ["-h", "--help"]},
+    pretty_exceptions_show_locals=False,
```

Comment:
I didn't find any easy way to write a test confirming change of this flag's value but I tested it manually.